src/test/java/com/hubspot/jinjava/EagerTest.java

```diff
@@ -478,7 +478,7 @@ public void itEvaluatesNonEagerSet() {
           .flatMap(eagerToken -> eagerToken.getUsedDeferredWords().stream())
           .collect(Collectors.toSet())
       )
-      .contains("deferred");
+      .isEmpty();
```

Comment:
These test changes are because now we only add words to `usedDeferredWords` if they aren't already `DeferredValues`. Here, `deferred` is already a deferred value so it won't be present in `usedDeferredWords`.